src/comp/comp.py

```diff
@@ -25,47 +28,82 @@ def __repr__(self):
 # whose name starts with 'D':
 print("Starts with D:")
 a = []
```

Comment:
Here's the syntax I was talking about:
```
a = [x.name for x in humans if x.name[0] == "D"]
```

I'll let you run with it so you can play with it on the other problems!